packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -147,6 +150,7 @@ class WebView extends StatefulWidget {
     this.gestureRecognizers,
     this.onPageStarted,
     this.onPageFinished,
+    this.onReceivedError,
```

Comment:
another nit: why onReceivedError and not onError/onResourceLoadError?

Reply:
It was the name of the method in `WebViewClient`. Same as `onPageStarted` and `onPageFinished`. It was mostly to keep the pattern of method name origins, but I think `WebResourceError` is more expressive. (or something similar)